site/src/pages/CreateTemplatePage/CreateTemplateForm.tsx

```diff
@@ -226,12 +234,25 @@ export const CreateTemplateForm: FC<CreateTemplateFormProps> = (props) => {
               }}
             />
           )}
+          {multiOrgExperimentEnabled && (
+            <OrganizationAutocomplete
+              {...getFieldHelpers("organization_id")}
+              value={selectedOrg}
+              onChange={(newValue) => {
+                setSelectedOrg(newValue);
+                return form.setFieldValue(
```

Comment:
since it's not part of the request body, we don't really need to track this as part of the form. I'd just stick to the `useState`.